utils/vercel-build/build.ts

```diff
@@ -0,0 +1,64 @@
+import fs from 'fs-extra';
+import { createGzip } from 'zlib';
+import tar from 'tar-fs';
+import { join } from 'path';
+import _frameworks, { Framework } from '@vercel/frameworks';
+// @ts-ignore
+import pipe from 'promisepipe';
+
+const ROOT_DIR = join(__dirname, '../..');
+const EXAMPLES_DIR = join(ROOT_DIR, 'examples');
+const PUBLIC_DIR = join(ROOT_DIR, 'public');
+
+async function main() {
+  // Start fresh
+  await fs.remove(PUBLIC_DIR);
+  await fs.mkdirp(PUBLIC_DIR);
+
+  // Output `frameworks.json`
+  const frameworks = (_frameworks as Framework[])
```

Comment:
This cast is no longer necessary

Reply:
<img width="546" alt="Screen Shot 2022-03-21 at 3 23 44 PM" src="https://user-images.githubusercontent.com/71256/159373103-a00fecab-39f8-405a-988a-1ed0899c8603.png">

Not so sure about that.

Reply:
Oh because sort() is mutating the array 😬